lstchain/reco/r0_to_dl1.py

```diff
@@ -182,9 +179,16 @@ def r0_to_dl1(input_filename = get_dataset_path('gamma_test_large.simtel.gz'),
 
     """
     if output_filename is None:
-        output_filename = (
-            'dl1_' + os.path.basename(input_filename).rsplit('.',1)[0] + '.h5'
-        )
+        if (input_filename.startswith('LST')):
```

Comment:
this might be a lot more elegant with using `pathlib.Path`.

e.g.

```python
>>>from pathlib import Path
>>>p = Path('run_001.simtel.gz')
>>>p.with_name('dl1_' + p.stem).with_suffix('.hdf5')  
```

Reply:
actually the `simtel` part is at the moment kept for the MC. I'm in favor of getting rid of it as well, anybody thinks we should keep it?

Reply:
Then you Can use p.name instead of p.stem

Reply:
done. 
I kept the `simtel` part for the time being, we can come back to this in the future if people are not happy.